internal/action/actions.go

```diff
@@ -160,6 +160,53 @@ func (h *BufPane) Center() bool {
 	return true
 }
 
+// CursorToViewTop moves the cursor to the top of the view,
+// offset by scrollmargin unless at the beginning or end of the file
+func (h *BufPane) CursorToViewTop() bool {
+	v := h.GetView()
+	h.Buf.ClearCursors()
+	scrollmargin := int(h.Buf.Settings["scrollmargin"].(float64))
+	bStart := display.SLoc{0, 0}
+	if h.Diff(v.StartLine, bStart) == 0 {
```

Comment:
It is the same condition as `v.StartLine == bStart` (the distance between two different visual lines cannot possibly be non-zero), so using `Diff()` is an overkill in this case.

Reply:
Fixed.